### DISCUSSION TITLE
Does this work in oconjunction with gitsigns plugin?

### DISCUSSION THREAD

**Discussion by molleweide** [tanvirtin/vgit.nvim#158] [Q&A] [answered]
Hi there,  just a quick question as the title sais - does it work with gitsigns at the same time?

**Comment by tanvirtin** [comment, marked as answer]
You should be able to use Gitsigns independent of this plugin afaik. To get a better VGit experience I would not recommend it, as some features are coupled with each other.

You can turn some similar features such as hunk signs and line blame off as you please from the settings.

For example:

```lua
require('vgit').setup({ settings = { live_gutter = { enabled = false } } })
```
